### PR TITLE
Sparkbox Sprint 16

### PR DESCRIPTION
**Pull request by silverli**
## Summary
Changes and stories addressed in Sparkbox Sprint 16:

- Traditional Theme work to get styles based on Colorful for All Themes | STN-337 #579 
-  Traditional Utility Class Work | STN-329 #588 
- QA Feedback: Sprint 15 | STN-319 #592 
- Minor Behat docs update | STN-327 #589 
- Traditional Theme Sweeping Elements Update | STN-344 #585 
- Humsci Basic Theme Spacing System / Traditional Theme Spacing Updates | STN-344 #586 
- Basic Page Default Hero Width Config Change | STN-295  #594 
- Traditional Theme Vertical Linked Card | STN-345 #591 
- Traditional Theme Date Stacked Card Tile | STN-347 #593 
- Traditional Theme Hero | STN-343 #595 

### TASK LIST:
- [x] Add compiled CSS & JS
- [x] Ensure Code Climate passes against the `develop` branch. Fix any errors. https://codeclimate.com/github/SU-HSDO/suhumsci/pull/596
- [x] Assign to Mike for review.

## Need Review By (Date)
5/12

## Urgency
high

## Steps to Test
1. `npm run test` should pass in the root folder
2. If you are interested in the testing steps for associated PRs, please take a look at the individual PR's description and "Steps to Test" section.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/components/_views-exposed-form.scss [hunk 1]
```diff
@@ -1,31 +1,51 @@
-.hb-layout-builder-main-content .views-exposed-form {
+.hb-three-column__main .views-exposed-form,
+.hb-three-column__sidebar-1 .views-exposed-form {
   display: flex;
-  align-items: flex-end;
+  align-items: flex-start;
```
Comment (bryanbraun) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_views-exposed-form.scss:4]: It looks like this is impacting the exposed form on the news page...
image

File: docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss [hunk 2]
```diff
@@ -177,67 +211,11 @@ blockquote:nth-child(n) {
     @include hb-blockquote;
 
     .hb-local-footer--dark & {
-      @include hb-colorful {
-        @include hb-global-color('color', 'gray-light');
-      }
+      @include hb-global-color('color', 'gray-light');
```
Comment (bryanbraun) [inline docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss:214]: This change didn't cause it, but I'm seeing an edge-case where non-p-tag dark-footer content isn't getting style overrides:

image

I wonder if now is a good time to put in a quick fix?

**Review by bryanbraun — COMMENTED** [review]
The code looks good... I'm currently looking into a couple of visual things (see comments).

File: docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss (hunk 2 above)
Comment (silverli) [reply docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss:214]: @bryanbraun I think it's worth checking if this is an actual thing H&S would want to do, it does seem a bit strange to have bullets in a blockquote. If they do want this behavior, we can add something the Q&A card.

File: docroot/themes/humsci/humsci_basic/src/scss/components/_views-exposed-form.scss (hunk 1 above)
Comment (silverli) [reply docroot/themes/humsci/humsci_basic/src/scss/components/_views-exposed-form.scss:4]: fixed!

**Review by bryanbraun — APPROVED** [review]
Looks great!

**Review by pookmish — APPROVED** [review]

**Pull request merged**